socorro/external/es/data/super_search_fields.json

```diff
@@ -3787,5 +3787,24 @@
         "is_exposed": true,
         "form_field_choices": [],
         "description": "'extension' if a WebExtension, 'web' or missing otherwise. This is only set for content crashes."
+    },
+    "telemetry_environment": {
+       "default_value": null,
+       "is_mandatory": false,
+       "description": "A field containing the entire Telemetry Environment, as sent with crash pings to Telemetry.",
+       "is_returned": true,
+       "query_type": "string",
+       "permissions_needed": [],
+       "storage_mapping": {
+          "index": "not_analyzed",
+          "type": "string"
```

Comment:
The type "string" is deprecated in ES >= 5, you want to use ``text`` for analyzed content and ``keyword`` for non-analyzed content instead.